### DISCUSSION TITLE
GitHub Profile: Is Having Many Inactive Repositories a Concern?

### DISCUSSION THREAD

**Discussion by Josekariz** [community/community#131110] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Does having numerous inactive repositories on GitHub affect a developer's professional image?

> Does the presence of many inactive repositories impact how developers are perceived.

> Should I and devs prioritize completing and deploying projects over creating new repositories for learning or experimentation?

**Comment by vishxly** [comment, marked as answer]
Having many inactive repositories can signal ongoing learning but might also suggest a lack of follow-through. Striking a balance is key; showcasing completed projects alongside experimental ones highlights both dedication and a commitment to growth.

**Reply by ebndev** [reply]
Hi @yadavvshall, thanks for taking the time to answer!

@Josekariz welcome to GitHub Discussions! I see that you're new here. Please feel encouraged to introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and connect with other users there! 👋
